docs/how-tos/use-multi-format-reader.md

```diff
@@ -0,0 +1,399 @@
+
+## Getting started
+
+**Note: You can find all of the data and the developed python scripts [here](https://download-directory.github.io/?url=https://github.com/FAIRmat-NFDI/pynxtools/tree/master/examples/mock-data-reader).**
+
+Here, we will implement a reader called `MyDataReader` that builds on the `MultiFormatReader`. `MyDataReader` is an example for a reader that can read HDF5 data from a specific technology-partner data set, as well as additional metadata from am electronic lab notebook (in YAML format).
+
+For demonstration purposess, we will work with a very simple mock application definition:
+
+```xml
+<definition xmlns="http://definition.nexusformat.org/nxdl/3.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" category="application" type="group" name="NXsimple" extends="NXobject" xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd">
+    <doc>
+         Mock application definition.
+    </doc>
+    <group type="NXentry">
+        <field name="definition">
+            <attribute name="version"/>
+            <enumeration>
+                <item value="NXsimple"/>
+            </enumeration>
+        </field>
+        <field name="title"/>
+        <group type="NXuser" recommended="true">
+            <field name="name">
+                <doc>
+                     Name of the user.
+                </doc>
+            </field>
+            <field name="address" recommended="true">
+                <doc>
+                     Name of the affiliation of the user.
+                </doc>
+            </field>
+        </group>
+        <group type="NXinstrument">
+            <doc>
+                 Description of the instrument and its individual parts.
+            </doc>
+            <attribute name="version">
+                <doc>
+                     Version of the instrument.
+                </doc>
+            </attribute>
+            <group type="NXdetector">
+                <field name="count_time" type="NX_NUMBER" units="NX_TIME" recommended="true">
+                    <doc>
+                         Elapsed actual counting time
+                    </doc>
+                </field>
+            </group>
+        </group>
+        <group name="sample" type="NXsample">
+            <field name="name"/>
+            <field name="physical_form" recommended="true"/>
+            <field name="temperature" type="NX_FLOAT" recommended="true" units="NX_TEMPERATURE"/>
+        </group>
+        <group name="data" type="NXdata">
+            <doc>
+                 The default NXdata group containing a view on the measured data.
+            </doc>
+        </group>
+    </group>
+</definition>
+```
+
+The NXDL requires a user, some sample information, some instrument metadata, and the measured data to be written. Some groups, fields, and attributes are strictly required, but just recommended.

```

Comment:
Explain where to copy this file to make it be recognized

Reply:
added